src/test/transform.ts

```diff
@@ -107,8 +108,10 @@ export function wrapFunctionWithLocation<A extends any[], R>(func: (location: Lo
     const oldPrepareStackTrace = Error.prepareStackTrace;
     Error.prepareStackTrace = (error, stackFrames) => {
       const frame: NodeJS.CallSite = sourceMapSupport.wrapCallSite(stackFrames[1]);
+      const fileName = frame.getFileName();
+      const file = (fileName && fileName.startsWith('file://')) ? url.fileURLToPath(fileName) : fileName;
```

Comment:
node error stacks for modules use `file://` urls instead of paths.

Reply:
(Would that be a reason to add it as a comment? - not sure about the comment rule here yet)